cypress/e2e/assignmentpractice.cy.js

```diff
@@ -0,0 +1,46 @@
+///<reference types='cypress'/>
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // return false to prevent Cypress from failing the test
+    return false
+  })
```

Comment:
try to add this in `command.js` file only once, it will work for every page. No need to add in every test file.